generators/generator-constants.js

```diff
@@ -27,9 +27,9 @@ const DOCKER_MONGODB = 'mongo:4.0.12';
 const DOCKER_COUCHBASE = 'couchbase:6.0.0';
 const DOCKER_CASSANDRA = 'cassandra:3.11.4';
 const DOCKER_MSSQL = 'microsoft/mssql-server-linux:latest';
-const DOCKER_HAZELCAST_MANAGEMENT_CENTER = 'hazelcast/management-center:3.12.3';
+const DOCKER_HAZELCAST_MANAGEMENT_CENTER = 'hazelcast/management-center:3.12.4';
 const DOCKER_MEMCACHED = 'memcached:1.5.16-alpine';
-const DOCKER_KEYCLOAK = 'jboss/keycloak:6.0.1';
+const DOCKER_KEYCLOAK = 'jboss/keycloak:7.0.0';
```

Comment:
we should also try to align `realm` and `user` configurations along with upgrades.

Reply:
@vishal423 In which repo?

Reply:
@vishal423 You mean here: https://github.com/jhipster/jhipster-registry/blob/master/src/main/docker/keycloak.yml

Reply:
@DanielFran, we should upgrade keycloak configurations like done in this PR #9635

Reply:
@vishal423 Got it! I will revert the update here and add a note to refer the necessary changes.
Will open a new Issue for that...
Thanks